gnucash/import-export/qif-imp/dialog-account-picker.c

```diff
@@ -318,6 +326,40 @@ gnc_ui_qif_account_picker_map_cb(GtkWidget * w, gpointer user_data)
 }
 
 
+/****************************************************************
+ * dialog_response_cb
+ *
+ ****************************************************************/
+static void
+dialog_response_cb (GtkDialog *dialog, gint response_id, gpointer user_data)
```

Comment:
An error dialog is always disruptive. Can we avoid it ? For example can we disable the Ok button when a placeholder is selected ? Potentially with a status message somewhere indicating why the Ok button was disabled.

Reply:
I just copied what was similar in the OFX importer I think but can have another look at it before pushing.